src/rest/runners.rs

```diff
@@ -24,7 +24,9 @@ use crate::model::{GitLabRunner, GitLabRunnerConfig};
 )]
 #[tracing::instrument(skip(pool))]
 pub async fn create(
-    State(AppState { pool, config_path }): State<AppState>,
+    State(AppState {
+        pool, config_path, ..
```

Comment:
what does the ` ..` do? I assume this is not a typo :D For educational purposes.

Reply:
"I want to destructure the `AppState` object but I only need the fields `pool` and `config_path`; omit the other fields"

https://doc.rust-lang.org/book/ch18-03-pattern-syntax.html#ignoring-values-in-a-pattern